rtos/mbed_lib.json

```diff
@@ -3,5 +3,22 @@
     "config": {
         "present": 1
     },
-    "macros": ["_RTE_"]
+    "macros": ["_RTE_"],
+    "target_overrides": {
+        "*": {
+            "target.boot-stack-size": "0x400"
+        },
+	    "MCU_NRF51": {
+	        "target.boot-stack-size": "0x800"
```

Comment:
Will it be good to move these overrides in targets.json instead?

Reply:
The size in targets.json is the size for mbed 2  - `0x1000`, while rtos/mbed_lib.json is for mbed 5 - `0x400`. In the case of the nrf5x the interrupt stack needs to be customized for mbed 5 only so it is set to `0x800` here.